demos/embeddings/deep-graph-infomax-cora.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>I know you mentioned choosing a lower number of epochs to make this run faster, but this kinda looks like the loss is still decreasing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
bupmed epochs upto 100 and its near convergence now.